operations/work-experience/complete-build/src/quiz.js

```diff
@@ -1,49 +1,43 @@
-const quizObject = {
-  quiz: [
-    {
-      question: "What is the capital of France?",
-      options: ["Paris", "London", "Madrid"],
-      answer: "Paris",
-    },
-    {
-      question: "What is the largest country in the world?",
-      options: ["Russia", "China", "USA"],
-      answer: "Russia",
-    },
-    {
-      question: "What is the chemical symbol for gold?",
-      options: ["Au", "Ag", "Hg"],
-      answer: "Au",
-    },
-  ],
-};
+import quiz from "./data/data.json" assert { type: "json" };
 
+//Initialization of a question count and the score to 0
 let questionsCount = 0;
 let score = 0;
+
+//Store the html elements into variables to use them in the different functions
 const title = document.getElementById("title");
 const buttonContainer = document.getElementById("buttonContainer");
 const questionContainer = document.getElementById("question");
 const answersContainer = document.getElementById("answers");
 
+//Display the first page of the quiz
 function buildQuiz() {
   title.innerText = `Welcome to the Online Quiz Project`;
-  buttonContainer.innerHTML = `<button onclick='questionPage()'>Let's go!</button>`;
+  buttonContainer.innerHTML = `<button>Let's go!</button>`;
+  buttonContainer.onclick = function () {
```

Comment:
This is a bit misleading: the `onclick` event is applied to the container of the button instead of the button itself.

I would rewrite this (and the other instances below) like the following:

```
const button = document.createElement("button");
button.innerHTML = "Let's go!";
button.onClick = questionPage;

buttonContainer.appendChild(button);
```